src/jsonrpc.rs

```diff
@@ -0,0 +1,62 @@
+use serde::{Deserialize, Serialize};
+use std::fmt::Debug;
+
+#[derive(Debug, PartialEq, Clone, Hash, Eq, Deserialize, Serialize, PartialOrd, Ord)]
+#[serde(deny_unknown_fields)]
+#[serde(untagged)]
```

Comment:
I think we should also add `#[serde(rename_all = "camelCase")]` to all values. We did that in the XDR lib, which is used in JSON in the CLI, so it'd keep both using the same form.